src/stores/snapshot-slice.ts

```diff
@@ -379,6 +391,11 @@ window.ipcListener.subscribe( 'snapshot-error', ( event, payload ) => {
 			title: __( 'Updating preview site failed' ),
 			message: payload.data.message,
 		} );
+	} else if ( operation.type === 'delete' ) {
+		getIpcApi().showErrorMessageBox( {
+			title: __( 'Deleting preview site failed' ),
+			message: payload.data.message,
+		} );
```

Comment:
This was another omission where I had overlooked displaying an error message box when a delete operation failed.